pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -167,20 +170,23 @@ func NewController(options *ControllerOptions) (*Controller, error) {
 			return nil, err
 		}
 
-		if err = qm.SyncMonitors(resources); err != nil {
+		if err = qm.SyncMonitors(ctx, resources); err != nil {
 			utilruntime.HandleError(fmt.Errorf("initial monitor sync has error: %v", err))
 		}
 
 		// only start quota once all informers synced
-		rq.informerSyncedFuncs = append(rq.informerSyncedFuncs, qm.IsSynced)
+		rq.informerSyncedFuncs = append(rq.informerSyncedFuncs, func() bool {
+			return qm.IsSynced(ctx)
+		})
 	}
 
 	return rq, nil
 }
 
 // enqueueAll is called at the fullResyncPeriod interval to force a full recalculation of quota usage statistics
-func (rq *Controller) enqueueAll() {
-	defer klog.V(4).Infof("Resource quota controller queued all resource quota for full calculation of usage")
+func (rq *Controller) enqueueAll(ctx context.Context) {
+	logger := klog.FromContext(ctx)
```

Comment:
Should we add a name to the logger?
Reference: https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/3077-contextual-logging#transition 
Quoting from enhancement guide:
Incorrect:
```go
func foo(ctx context.Context) {
   logger := klog.FromContext(ctx)
   ctx = klog.NewContext(ctx, logger.WithName("foo"))
   doSomething(ctx)
   // BUG: does not include WithName("foo")
   logger.Info("Done")
}
```

Correct:
```go
func foo(ctx context.Context) {
   logger := klog.FromContext(ctx).WithName("foo")
   ctx = klog.NewContext(ctx, logger)
   doSomething(ctx)
   logger.Info("Done")
}
```

Reply:
From the [Slack thread](https://kubernetes.slack.com/archives/C020CCMUEAX/p1668087453736599), there isn't a pitfall issue with this code because it isn't setting a name and then missing updating the logger in the context.

The actual question is: should we set a name?

I don't feel that it's necessary, but happy to do whatever we collectively decide.

Reply:
Update: Adding a name to logger is a choice of developer (and their developer practices), which means this change is not necessary for migration to contextual logging

Reply:
Discussed this a bit more and for kube-controller-manager adding a different name for each controller makes sense. Let's do it in app/core.go. See https://github.com/kubernetes/kubernetes/pull/113916#discussion_r1025594468

Reply:
Updated, PTAL!